tests/assets/js/some-long-task.js

```diff
@@ -0,0 +1,5 @@
+function someLongTask() {
+	const startTime = Date.now();
+	while (Date.now() - startTime < 60);  // long tasks are those >50ms which blocks main thread
```

Comment:
Is this definition of a long task the same for all browsers?

Reply:
This https://inp-demo.glitch.me/ is a good demo to play around with INP and they used the same approach. I believe it should be good enough.

Reply:
@patrickhousley yes, it's in the official definition of this timing API.